src/testruns.py

```diff
@@ -45,6 +45,6 @@ def main(**kwargs):
 print(tswizzle(31))
 
 #Get a file from this directory
-# with open('testfile.txt', 'r') as f:
```

Comment:
Could you use more descriptive instead of single letter variable names? This will help make the code more readable.

Reply:
also select the line below the code you want to comment on when reviewing, that way it shows up in the PR discussion

Reply:
Fixed in https://github.com/brandonbush/test-git-commands/pull/5/commits/774de643729f011abbfab48781d2188dab280252